src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
+  <string name="syncing_format">Syncing %s…</string>
   <string name="tap_to_install_description">Tap to install.</string>
   <string name="theme">Theme</string>
   <string name="uninstall">Uninstall</string>
   <string name="unknown">Unknown</string>
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    application from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest upgrades to unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about new applications</string>
+  <string name="update_notifications_summary">Show a notification when new versions are available</string>
   <string name="updates">Updates</string>
   <string name="updates_empty_summary">All applications up to date</string>
   <string name="username">Username</string>
-  <string name="username_is_not_specified">Username is not specified</string>
-  <string name="validation_index_error_description">Failed to validate the index.</string>
-  <string name="validation_package_error_description_format">Failed to validate the package: %s.</string>
-  <string name="validation_error_hash_lower">invalid hash</string>
-  <string name="validation_error_integrity_lower">integrity check failed</string>
-  <string name="validation_error_permissions_lower">missing permissions</string>
-  <string name="validation_error_signature_lower">invalid signature</string>
-  <string name="validation_error_structure_lower">invalid file format</string>
+  <string name="username_is_not_specified">Username not specified</string>
+  <string name="validation_index_error_description">Could not validate the index.</string>
+  <string name="validation_package_error_description_format">Could not validate the package: %s.</string>
+  <string name="validation_error_hash_lower">Invalid checksum</string>
+  <string name="validation_error_integrity_lower">Could not check integrity</string>
+  <string name="validation_error_permissions_lower">Add reqiored permissions</string>
+  <string name="validation_error_signature_lower">Invalid signature</string>
+  <string name="validation_error_structure_lower">Invalid file format</string>
```

Comment:
"*_lower" means that string should be started from a small letter. Don't change this.

Reply:
"Could not check integrity" is also not good here, because after formatting if will look like that:

> Could not validate the package: Could not check integrity

This is why I'm using "integrity check failed"

Reply:
If it is always after a colon, it is better to use "*_Upper". If not, chopping sentences and splicing them produces errors and fails in translation.
Even in sentences the context is often lost, like I did.
How about: "Could not validate the integrity of the package."

Reply:
Yes, always after a colon. Why uppercase is better?

Reply:
Because then it doesn't get translated as if it is a spliced sentence, or erroneously starting with a small versal.

Reply:
I don't think it's right to make this strange capitalization just because it's hard to explain why the first letter is lower case. Afaik, weblate supports comments (or descriptions?) for each lines, I will explain that.

Reply:
It is allowed to follow a colon with a big versal, provided what is behind the colon is a full sentence. The strange convention is to do both, which no other language does to my knowledge. : Bigversal produces fewer errors in translation. You can put it in the description, and the same people that get it wrong will skip reading the description.

Reply:
In the worst case in their translation the followed by colon sentence will start from uppercase in their language (which someone else can fix afterwards). I don't see any problems with it. I'm using English version, and I prefer the lowercase.

Reply:
OK, small it is.